bin/emqx

```diff
@@ -248,8 +248,9 @@ fi
 
 # Extract the target node name from node.args
 if [ -z "$NAME_ARG" ]; then
-    if [ ! -z "$EMQX_NODE_NAME" ]; then
-        NODENAME="$EMQX_NODE_NAME"
+    if [ ! -z "$EMQX_NODE__NAME" ]; then
```

Comment:
`EMQX_NODE_NAME` is very extensively used and likely well received by many users. 
I'd recommend check `EMQX_NODE_NAME` if `EMQX_NODE__NAME` is not given.

Reply:
or `EMQX_NODE_NAME=${EMQX_NODE_NAME:-${EMQX_NODE__NAME:-}}`